Source/Protocol/Row.swift

```diff
@@ -40,12 +40,14 @@ public protocol Row {
 
 }
 
-
 extension Row {
-
   /// Returns true iff `lhs` and `rhs` have equal titles and subtitles.
   public static func == (lhs: Self, rhs: Self) -> Bool {
     return lhs.title == rhs.title && lhs.subtitle == rhs.subtitle
   }
+}
 
+/// Returns true iff `lhs` and `rhs` have equal titles and subtitles.
+public func == (lhs: Row, rhs: Row) -> Bool {
+  return lhs.title == rhs.title && lhs.subtitle == rhs.subtitle
```

Comment:
Well my reason for doing this was to add a slight bit more dynamism to the library, I used it to traverse the tableContents matching a Row item to get the indexPath, use that to get the cell, and then call `customize()` from the row item to that cell.

What might make it even better would be to simply declare the Row protocol as a class protocol. That would allow for identity checks which would be even better.

Reply:
I think this global `==` function would be better declared in your project's scope where you need it. Having the `Row` as a class protocol sounds like a good solution to me. 👍